tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -171,3 +173,109 @@ Feature: collaboration (wopi)
       | app-endpoint                                     |
       | /app/open?file_id=<<FILEID>>&app_name=FakeOffice |
       | /app/open?file_id=<<FILEID>>                     |
+
+
+  Scenario Outline: open file with .odt extension in web
+    Given user "Alice" has uploaded file "filesForUpload/simple.odt" to "simple.odt"
+    And we save it into "FILEID"
```

Comment:
this PR needs others to be merged first

Reply:
Yes please. Waiting for those to be  merged